docs/faq.md

```diff
@@ -17,6 +17,15 @@ Many browsers support this standard and should work fine, but some have not
 implemented it yet (mobile browsers lag behind on this, in
 particular).
 
+## Why does Firefox Send require JavaScript?
+
+- We use JavaScript to encrypt and decrypt files locally on the client instead of the server.
+- We use JavaScript to render the user interface of Send.
+- We use JavaScript to manage translate the website into [various different languages](https://github.com/mozilla/send#localization).
+- We use JavaScript to collect data to help us improve Send in accordance with our [Terms &amp; Privacy](https://send.firefox.com/legal).
+
+Since Send is an open source project, you can see all of the cool ways we use JavaScript by [examining our code](https://github.com/mozilla/send/).
```

Comment:
Copied the general concept/text from [**mozilla/testpilot** docs/faq.md](https://github.com/mozilla/testpilot/blob/master/docs/faq.md#why-does-test-pilot-require-javascript), but could probably use a bit of tweaking.